fms/utils/gptq.py

```diff
@@ -0,0 +1,172 @@
+from dataclasses import dataclass
+from typing import Mapping, Any
+import torch
+import torch.nn as nn
+from fms.utils.config import ModelConfig
+from fms.modules.linear import (
+    register_linear_type_to_module_map,
+    register_linear_type_to_sharding_map,
+    shard_base_linear,
+)
+
+try:
+    from auto_gptq.utils.import_utils import dynamically_import_QuantLinear
+    IS_AUTOGPTQ_AVAILABLE=True
+except:
+    IS_AUTOGPTQ_AVAILABLE=False
+
+
+# simplified from AutoGPTQ quantization config
+# see: https://github.com/AutoGPTQ/AutoGPTQ/blob/caf343b1826301c15f90e2e119cabd0347acfcdf/auto_gptq/quantization/config.py#L60
+@dataclass
+class GPTQConfig(ModelConfig):
```

Comment:
Renaming this to `GPTQLinearConfig` so it's more specific, in contrast to `Fp8LinearConfig`